### DISCUSSION TITLE
Silent DC

### DISCUSSION THREAD

**Discussion by Pitti999** [ORelio/Minecraft-Console-Client#1948] [Q&A]
I've been work a while on getting this MCC to work... But i'm close to a point, where im gonna give up. Nothing seems to work.

My problem is, when connecting to the server, the play stays on there for like 28 sec, before time out.

The log with debug on, gives me nothing to go on.

### Log started at 2022-02-18 12:58:18 ###
Version is supported.
Logging in...
[DEBUG] Crypto keys & hash generated.
Checking Session...
Server was successfully joined.
Type '/quit' to leave the server.
"PLAYER" has connected
Connection has been lost.

So what have I done?

First of all I installed Ubuntu server (also tried on debian - same result) on my Pi and my server at home.

Then I follow the mono guide, step by step https://www.mono-project.com/download/stable/#download-lin - get no errors while installing it.

Step 3 - Get the MinecraftClient.exe.... This is a pain. When runinng `msbuild MinecraftClient.csproj` it returns Microsoft (R) Build `Engine version 16.6.0 for Mono`
`Copyright (C) Microsoft Corporation. All rights reserved.`

`MSBUILD : error MSB1009: Project file does not exist.`
`Switch: MinecraftClient.csproj`

Therefor I tried running `wget https://github.com/MCCTeam/Minecraft-Console-Client/releases/download/20220206-9/MinecraftClient.exe` wich goes well. I type in my login info, and now we're back to the start of this post. DC after ~28 sec.

My question is: What am I doing wrong here, and how do I fix it?

**Comment by Pitti999** [comment]
Got it, thanks...

**Comment by ORelio** [comment]
Just putting this in case anyone else has this issue:
Try making sure you are in the correct folder containing `MinecraftClient.csproj` by using `ls` and `cd <name of folder>`
